### PR TITLE
AutoRelayer: js sdk includes MessageKey serde

### PR DESCRIPTION
**Pull request by JoeHowarth**

### CODE REVIEW

File: sdk/js/src/relayer/structs.ts [hunk 1]
```diff
@@ -41,12 +41,31 @@ export function parseRefundStatus(index: number) {
...
+}
+
 export interface VaaKey {
   chainId: number;
   emitterAddress: Buffer;
   sequence: BigNumber;
 }
 
+export interface CCTPKey {
+  domain: number;
```
Comment (solanoepalacio) [inline sdk/js/src/relayer/structs.ts:60]: just curious, what does domain represent in this context?

File: sdk/js/src/relayer/relayer/info.ts [hunk 2]
```diff
@@ -219,16 +222,23 @@ export function stringifyWormholeRelayerInfo(
...
+              vaaKey.chainId ? printChain(vaaKey.chainId) : ""
+            }, with emitter address ${vaaKey.emitterAddress?.toString(
+              "hex"
+            )} and sequence number ${vaaKey.sequence}`;
+          } else if (msgKey.keyType == KeyType.CCTP) {
+            const cctpKey = parseCCTPKey(msgKey.key);
+            result += `(CCTP ${i}): `;
+            result += `Transfer from cctp domain ${printChain(cctpKey.domain)}`;
+            result += `, with nonce ${cctpKey.nonce}`;
+          }
```
Comment (derpy-duck) [inline sdk/js/src/relayer/relayer/info.ts:241]: maybe a default case - print out 'Message of key type ${msgKey.keyType} with this identifier'

File: sdk/js/src/relayer/structs.ts (hunk 1 above)
Comment (JoeHowarth) [reply sdk/js/src/relayer/structs.ts:60]: Domain is Circle's version of chainId

File: sdk/js/src/relayer/relayer/info.ts (hunk 2 above)
Comment (JoeHowarth) [reply sdk/js/src/relayer/relayer/info.ts:243]: Done

**Pull request merged**